src/Microsoft.AspNetCore.Http.Connections.Common/AvailableTransport.cs

```diff
@@ -3,7 +3,7 @@
 
 using System.Collections.Generic;
 
-namespace Microsoft.AspNetCore.Http.Connections.Internal
+namespace Microsoft.AspNetCore.Http.Connections
```

Comment:
You're going to have to `IList<string>` the `TransferFormats` property if this is going to be public...

Reply:
Same in NegotiationResponse